docs/pages/installation/gitpod.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Gitpod
+permalink: installation/gitpod
+type: installation
+category: kubernetes
+redirect_from:
+- installation/platforms/gitpod
+display-title: "false"
+language: en
+list: include
+image: /assets/img/platforms/gitpod.png
+abstract: Build and contribute to Meshery using Gitpod
+---
+
+<h1>Quick Start with {{ page.title }} <img src="{{ page.image }}" style="width:40px;height:40px;" /></h1>
+
+Use Minikube in Gitpod to setup your development environment for Meshery.
+
+<div class="prereqs"><p><strong style="font-size: 20px;">Prerequisites</strong> </p> 
+  <ol>
+    <li>Make an account on <a href="https://www.gitpod.io/" class="meshery-light">Gitpod</a> and add the <a href="https://chromewebstore.google.com/detail/gitpod/dodmmooeoklaejobgleioelladacbeki" class="meshery-light">Gitpod extension</a> to your browser.</li>
+     <li>Install the Meshery command line client, <a href="{{ site.baseurl }}/installation/mesheryctl" class="meshery-light">mesheryctl</a>.</li>
+  </ol>
+</div>
+
+## Available Deployment Methods
+
+- [In-cluster Installation](#in-cluster-installation)
+  - [Preflight Checks](#preflight-checks)
+    - [Preflight: Cluster Connectivity](#preflight-cluster-connectivity)
+  - [Installation: Using `mesheryctl`](#installation-using-mesheryctl)
+  - [Installation: Using Helm](#installation-using-helm)
+  - [Installation: Manual Steps](#installation-manual-steps)
+  - [Access Meshery UI](#access-meshery-ui)
+
+# In-cluster Installation
+
+Follow the steps below to install Meshery in your Minikube cluster.
+
+## Preflight Checks
+
+Read through the following considerations prior to deploying Meshery on Minikube.
+
+### Preflight: Cluster Connectivity
+
+
+You can develop and run Meshery in a Gitpod using your choice of tool:
+
+- A command shell, via an SSH connection initiated using GitHub CLI.
+- One of the JetBrains IDEs, via the JetBrains Gateway.
+- The Visual Studio Code desktop application.
+- A browser-based version of Visual Studio Code.
+#### Fork and Open in Gitpod:
+- Open your forked Meshery repository on GitHub.
+- Click on the "Gitpod" button in the top right corner of the repository page.
```

Comment:
Mention that the Gitpod button is only visible with the browser extension

Reply:
Hey @MonalikaPatnaik! You can also add the [link](https://chromewebstore.google.com/detail/gitpod/dodmmooeoklaejobgleioelladacbeki) to extension.

Link: https://chromewebstore.google.com/detail/gitpod/dodmmooeoklaejobgleioelladacbeki

Reply:
I have added to add the chrome extension for gitpod in the prerequisite only